src/style.css

```diff
@@ -0,0 +1,88 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  box-sizing: border-box;
+  height: 60px;
+  position: relative;
+  margin: 0;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav_list {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  margin: 0;
+  list-style-type: none;
```

Comment:
better to write
```suggestion
ul {
  list-style-type: none;
}
```

Reply:
Why is this better?

Reply:
you can have many `ul` on your page, better to write this rule once.

Reply:
I fix this.